style.css

```diff
@@ -0,0 +1,70 @@
+body {
+  width: 320px;
+  background-color: #e5e5e5;
+  margin: 50px;
+}
+
+.navbar {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+}
+
+.pic1 {
+  width: 111px;
+  height: 74px;
+}
+
+.menu {
+  width: 20px;
+  height: 14px;
+  margin-top: 19px;
+  margin-right: 19px;
+}
+
+.headline {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+h1 {
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 50px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 50px;
+  text-align: left;
+  margin: 30px 16px 20px 16px;
+  color: #1a2236;
+}
+
+p {
+  font-family: 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+  margin: 0 16px;
+  height: 126px;
+}
```

Comment:
- [ ] Kindly make the font size here bigger to match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/158731221-a215eee1-f308-4e4d-8272-664686346c4b.png) | ![image](https://user-images.githubusercontent.com/35535130/158731268-5001dd81-588a-4ec5-923a-a6895a4ddc1b.png) |

Reply:
resolved 👍